### PR TITLE
feat: add swith darkmode with js

### PR DESCRIPTION
**Pull request by fpasquet**
### Related story / issue:

- Zube: https://zube.io/20minutes/shuri/c/381
- Github issue: #50
- - Github issue: #187
- Screenshots?
image

image

### CODE REVIEW

**Review by bmillot — DISMISSED** [review]
LGTM

**Review by KysterMax — DISMISSED** [review]
LGTM

**Review by bmillot — APPROVED** [review]
LGTM

File: src/assets/scripts/theming.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+export const listennerOfSwitchDarkModeElement = (
+  switchDarkModeElement: null | HTMLButtonElement = document.querySelector<HTMLButtonElement>(
+    '.js-swith-dark-mode'
+  )
+): void => {
+  if (switchDarkModeElement) {
+    switchDarkModeElement.addEventListener('click', () => {
+      const currentTheme = document.documentElement.dataset.theme as ThemeType
+      const newTheme = !currentTheme || currentTheme === 'light' ? 'dark' : 'light'
```
[the remark this answers is not in the document]
Comment (JamesLaviron) [reply src/assets/scripts/theming.ts:37]: Was fine, nvm

**Review by JamesLaviron — APPROVED** [review]

**Pull request merged**